Ahmadreza-tahmasebi/c7.js

```diff
@@ -0,0 +1,124 @@
+function Product(itemName, count, price, isEnable=true) {
+    return({
+        "name" : itemName,
+        "count" : count,
+        "price" : price,
+        "isEnable" : isEnable
+    })
+}
+
+
+// 1
+function getAllCounts() {
+    var res = [];
+    for(var item of products){
+        console.log(`${item.name} : ${item.count}`);
+        res.push(`${item.name} : ${item.count}`);
+    }
+    return res;
+}
+
+// 2
+function newItem(name,count){
+    var item = Product(name, count, 0)
+    products.push(item)
+    return item;
+}
+
+// 3
+function sell(itemName){
+    for(var item of products){
+        if (item.name === itemName){
```

Comment:
Better to use `id` instead of `name` as it will ensure us that the item is unique.